django_admin_row_actions/admin.py

```diff
@@ -53,14 +40,14 @@ def to_dict(tool_name):
         items = []
 
         row_actions = self.get_row_actions(obj)
-        url_prefix = '{}/'.format(obj.pk if includePk else '')
+        url_prefix = "{obj.pk if includePk else ''}/"
```

Comment:
see comment on `Quotes`

Reply:
Resolved in [90cd571](https://github.com/DjangoAdminHackers/django-admin-row-actions/pull/28/commits/90cd571e14e08f567e7f3382262bf6eae39f457d).